Cargo.lock

```diff
@@ -4621,7 +4648,7 @@ dependencies = [
  "hermit-abi 0.3.0",
  "libc",
  "miniz_oxide",
- "object",
+ "object 0.30.3",
```

Comment:
Is depending on old version of `object` for `std` intentional?

Reply:
Yes, it's pointless to bump this unless backtrace is updated first.